src/operators/sequence.ml

```diff
@@ -152,7 +152,9 @@ let () =
   Lang.add_operator "merge_tracks"
     [ "", Lang.source_t k, None, None ]
     ~category:Lang.TrackProcessing
-    ~descr:"Merge consecutive tracks from the input source."
+    ~descr:"Merge consecutive tracks from the input source. They will be \
+            considered as one big track, so on_track() will not trigger \
```

Comment:
Great, thanks!

Reply:
One thing, please use backquotes for code such as
```suggestion
            considered as one big track, so `on_track()` will not trigger \
```

Reply:
Right, forgot this syntax ^^